indexer-app/src/cmr/indexer/data/concepts/tag.clj

```diff
@@ -27,3 +27,13 @@
      :originator-id-lowercase  (util/safe-lowercase originator-id)
      :tag-value-lowercase (when (string? data)
                             (str/lower-case data))}))
+
+(def earthdata-cloud-s3-tag
+  "The Earthdata cloud tag for s3 resources"
+  "gov.nasa.earthdata.cloud.s3")
+
+(defn has-cloud-s3-tag
+  "Looks through a list of tags and returns true if one of them is the
+  gov.nasa.earthdata.cloud.s3 tag"
+  [tags]
+  (some? (some #(= (:tag-key-lowercase %) earthdata-cloud-s3-tag) tags)))
```

Comment:
This is a predicate function, so its name should have a question mark. 

`(some? (some ...))` seems very similar to `(not (empty? ...))`, which is less idiomatic than `(seq x)`
Consider something like this:
```suggestion
(defn has-cloud-s3-tag?
  "Return true if collection contains gov.nasa.earthdata.cloud.s3 tag"
  [tags]
  (seq (filter #(= (:tag-key-lowercase %) earthdata-cloud-s3-tag) tags)))
```

Reply:
will try this out.

Reply:
returns nil and not false, so I'm inclined to go with the original.